pkg/apis/alerting_notifications/v0alpha1/types_ext.go

```diff
@@ -57,6 +58,17 @@ func (o *Receiver) SetAccessControl(action string) {
 	o.Annotations[AccessControlAnnotation(action)] = "true"
 }
 
+func (o *Receiver) ClearAccessControl() {
+	if o.Annotations == nil {
+		return
+	}
+	for k := range o.Annotations {
+		if strings.HasPrefix(k, fmt.Sprintf("%s%s/", InternalPrefix, "access")) {
+			delete(o.Annotations, k)
+		}
+	}
+}
```

Comment:
This seems to be used only in tests. I wonder if we can just add it as a function in the test utils?

Reply:
cb225b466d148b247ab25cfc0e036271cdcf7dd8